tests/test_backend.py

```diff
@@ -3624,6 +3629,27 @@ def test_conv2d_1_kernel_as_input(self):
                           [1., 1., 4.]], dtype=np.float32).reshape(_KERNEL3x3)
         self._conv_kernel_as_input_test(x_val, w_val)
 
+    @unittest.skipIf(tf_signal is None, reason="TF does not have submodule signal.")
```

Comment:
Can you make this check depend on the TF version instead of doing the whole try/catch import thing?

Reply:
Fixed.